Makefile

```diff
@@ -50,6 +51,9 @@ test-e2e-clickhouse:
 test-e2e-vertica:
 	go test $(GO_TEST_FLAGS) ./tests/vertica
 
+test-e2e-ydb:
+	go test $(GO_TEST_FLAGS) -parallel=1 ./tests/e2e -dialect=ydb
```

Comment:
Curious, why need to set `-parallel=1`?

Reply:
Unfortunately, ydb container requires a lot of resources, and on a small machine, high parallelism with running many containers can provoke out-of-memory errors or unexpected behaviour. Because db docker image created for tests only, not for production. In production we have a distributed cluster of ydb nodes and perfect ACID guarantees